docs/config/README.md

```diff
@@ -877,11 +877,20 @@ The module will be shown if any of the following conditions are met:
 
 ### Options
 
-| Variable   | Default        | Description                                            |
-| ---------- | -------------- | ------------------------------------------------------ |
-| `symbol`   | `"☕ "`         | The symbol used before displaying the version of Java. |
-| `style`    | `"dimmed red"` | The style for the module.                              |
-| `disabled` | `false`        | Disables the `java` module.                            |
+| Variable   | Default                             | Description                                     |
+| ---------- | ----------------------------------- | ----------------------------------------------- |
+| `format`   | `"via [$symbol $version]($style) "` | The format for the module.                      |
+| `symbol`   | `"☕"`                               | A format string representing the symbol of Java |
```

Comment:
```suggestion
| `format`   | `"via [$symbol$version]($style) "` | The format for the module.                      |
| `symbol`   | `"☕ "`                             | A format string representing the symbol of Java |
```
Well, this change looks reasonable to me, but it will break the config before the format string release. I think it should use exactly the same config as before.

Reply:
Arf... not ideal but I get it for backward compatibility. Let me change that.